index.html

```diff
@@ -234,52 +268,50 @@ <h1 class="about_myself_heading">About me</h1>
       </ul>
     </section>
 
-    <section class="contact_section">
-      <h3>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h3>
-
-      <form
-        action="https://formspree.io/f/xgedowvw"
-        method="POST"
-        class="contact_form"
-      >
-        <input
-          class="name"
-          type="text"
-          placeholder="Full name"
-          required
-          maxlength="30"
-        />
-        <hr class="horizontal_line" />
-        <input
-          class="email"
-          type="email"
-          placeholder="Email address"
-          required
-        />
-        <hr class="horizontal_line" />
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          cols="30"
-          rows="10"
-          maxlength="500"
-          placeholder="Enter text here"
-          required
-        ></textarea>
-
-        <button class="submit_button" type="submit">Get in touch</button>
-      </form>
-    </section>
-
-    <hr class="horizontal_line horizontal_line_3" />
-
     <footer class="footer">
-      <div class="social-media social-media-footer">
-        <ul class="social-media-links">
+      <section class="contact_section" id="contact">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+
+        <form
```

Comment:
- [ ] Good job making the name of the user a single field!  Now let's make the form fields a little bigger horizontally to make them look like the figma:

![image](https://user-images.githubusercontent.com/57421823/149436177-4df1d805-9669-4da4-999b-0fe833b70365.png)

The figma:

![image](https://user-images.githubusercontent.com/57421823/149436251-85094ce9-f19e-4bbe-ac7d-870462eaf24b.png)

That way, your form will be hugely more consistent! 💪